tilecloud_chain/__init__.py

```diff
@@ -838,11 +856,22 @@ def counter(self, size=False):
         self.imap(count)
         return count
 
-    def process(self, name=None):
-        if name is None:
-            name = self.layer.get('post_process')
-        if name is not None:
-            self.imap(Process(self.config['process'][name], self.options))
+    def process(self, name=None, key='post_process'):
+        if self.layer:
+            if name is None:
+                name = self.layer.get(key)
+            if name is not None:
+                self.imap(Process(self.config['process'][name], self.options))
+        else:
+            processes = {}
+            for lname, layer in self.layers.items():
+                name_ = name
+                if name_ is None:
```

Comment:
In practice, never, but since the old version of this method had a name argument, I'm reluctant to remove it and potentially break custom chains